prototipo_brain_lesion/diagnostico/templates/home.html

```diff
@@ -37,6 +37,24 @@ <h5>{{ user.name | upper }}</h5>
     <h5>soy doctor</h5>
   {% endif %}
 
+  <div class="row">
+    <div class="col-12 col-xl-12 d-flex justify-content-center align-items-center">
+      <h5>Instrucciones</h5>
+    </div>  
+  </div> 
+  <div class="row">
+     <div class="col-2 col-xl-2"></div>
+      <div class="col-8 col-xl-8 d-flex justify-content-center align-items-center">
+        
+        <p class="text-justify">Bienvenidos, esta aplicaciòn permite segmentar lesiones producto de ACV (Accidente Cerebrovascular) y determinar la clasificaciòn
+        de la enfermedad cerebrovascular como: ACV, Lacunar. Para realizar esto el usuario tendrà que cargar un  archivo de Resonancia Magnètica de formato .nii.gz
```

Comment:
`MCA o Lacunar`

Reply:
Change `Resonancia Magnètica de formato .nii.gz` for `MRI en formato NIfTI (.nii, .nii.gz)`